src/main/java/com/traveltime/sdk/dto/responses/geojson/Coordinates.java

```diff
@@ -0,0 +1,18 @@
+package com.traveltime.sdk.dto.responses.geojson;
```

Comment:
I think it's better to take it out from responses(com.traveltime.sdk.geojson or smith like that)

Reply:
Do you mean to reuse the com.traveltime.sdk.dto.common.Coordinates class? 
I was thinking about it, but I didn't see an easy way to serialize/deserialize a single class in two different way. I think we would need to introduce two separate methods in jsonUtils - to/from json + to/from geoJson  with two different Mappers. Is it worth it?

Reply:
Ok, I'll change the package